integrations/applications/google-workspace/resources/template.json

```diff
@@ -0,0 +1,95 @@
+{
+    "payload": {
+        "name": "Google Workspace",
+        "advancedConfigurations": {
+            "discoverableByEndUsers": true,
```

Comment:
```
"discoverableByEndUsers": true,
```

Is this expected behavior? By default, this application will appear for all users in their My Account.

Reply:
Let's keep this setting for now.